lucet-runtime/lucet-runtime-internals/src/alloc/mod.rs

```diff
@@ -373,6 +373,7 @@ impl Limits {
             self.globals_size,
             host_page_size(),
```

Comment:
I'm not sure if this guard page really guards anything particularly interesting (globals overflow? sigstack underflow should be impossible), but I'm not worried about the 4k here, so.